fluid-project/trivet
# feat: add Netlify CMS configuration and internationalization support

Adds:

- [x] Netlify CMS configuration for editing posts, pages, site data and site config
- [x] Multilingual support for all collections (including English and Français for demonstration purposes)
- [x] Translations for URL components and other strings (e.g. `/posts/my-post` &rarr; `/articles/mon-article`)
- [x] Localized RSS feeds for each language
- [x] Localization of site head metadata tags (e.g. description)
- [x] Localization of UIO

### Testing

1. Run locally (clone, `npm install`, `npm start`).
2. Verify that internationalized content is editable via http://localhost:8080/admin.
3. Try adding a new language as per the docs.

### Related Issues

- Resolves #48.
- Resolves #141.
